src/App.js

```diff
@@ -23,7 +24,8 @@ const router = createBrowserRouter(
     <Route path="/" element={<NavBar />}>
       <Route index element={<Home />} />
       <Route path="reviews" element={<Review />} />
-      <Route path="reviews/:review_id" element={<SingleReview />} />
+      <Route path="reviews/:category" element={<Review />} />
```

Comment:
If you're going to a parametric it should generally be a subsection of the  previous path. i.e. `/categories/:category`

Reply:
ok thanks will implement that now That actually caused me a lot of stress yesterday as the links were similar it stopped the functionality of whichever was underneath the other in the app